addons/web/static/src/js/views/kanban/kanban_record_quick_create.js

```diff
@@ -91,9 +91,18 @@ var RecordQuickCreate = Widget.extend({
 
         // destroy the quick create when the user clicks outside
         core.bus.on('click', this, this._onWindowClicked);
+        // added mousedown to check whether to ignore window click or not
+        document.body.addEventListener('mousedown', this._onBodyMousedown.bind(this));
 
         return this._super.apply(this, arguments);
     },
+    /**
+     * @override
+     */
+    destroy() {
+        this._super(...arguments);
+        document.body.removeEventListener('mousedown', this._onBodyMousedown, true);
```

Comment:
this doesn't work, you are removing an handler that doesn't exist (the one you registered is `this._onBodyMousedown.bind(this)` which is another reference).

It would work if you do, in start
```js
this._onBodyMousedown = this._onBodyMousedown.bind(this);
document.body.addEventListener('mousedown', this._onBodyMousedown);
```

Reply:
Hello @aab-odoo 

Changes Done.

Thanks